docs/guides/algorithmiq-tem.ipynb

```diff
@@ -227,12 +273,12 @@
    "source": [
     "## Outputs\n",
     "\n",
-    "A Qiskit [PrimitiveResults](/api/qiskit/qiskit.primitives.PrimitiveResult) containing the TEM mitigated result. The result for each PUB is returned as a [PubResult](/api/qiskit/qiskit.primitives.PubResult) containing the following fields:\n",
+    "A Qiskit [PrimitiveResults](https://docs.quantum.ibm.com/api/qiskit/qiskit.primitives.PrimitiveResult) containing the TEM mitigated result. The result for each PUB is returned as a [PubResult](https://docs.quantum.ibm.com/api/qiskit/qiskit.primitives.PubResult) containing the following fields:\n",
     " \n",
     "\n",
     "Name |Type | Description\n",
     "-- | -- | --\n",
-    "data | DataBin | A Qiskit [DataBin](/api/qiskit/qiskit.primitives.DataBin) containing the TEM-mitigated observable and its standard error. The DataBin has the following fields: <ul><li>`evs`: The TEM-mitigated observable value.</li><li>`stds`: The standard error of the TEM-mitigated observable.</li></ul>\n",
+    "data | DataBin | A Qiskit [DataBin](https://docs.quantum.ibm.com/api/qiskit/qiskit.primitives.DataBin) containing the TEM mitigated observable and its standard error. The DataBin has the following fields: <ul><li>`evs`: The TEM mitigated observable value.</li><li>`stds`: The standard error of the TEM mitigated observable.</li></ul>\n",
```

Comment:
@abbycross, I think this be a relative link, right?

Reply:
Reverted to relative link